airflow/operators/python.py

```diff
@@ -26,6 +26,11 @@
 from textwrap import dedent
 from typing import Any, Callable, Dict, Iterable, List, Optional, Union
 
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
```

Comment:
There’s `airflow.utils.typing_compat` for this.

Reply:
Very cool. Thanks for the pro tip!